Makefile

```diff
@@ -36,15 +36,14 @@ python/rikai/spark/sql/generated/RikaiModelSchemaParser.py: src/main/antlr4/org/
 
 lint:
 	sbt scalafmtCheckAll
-	black -l 79 --check python contrib/
+	black --check python contrib experimental
 	pycodestyle python contrib/
 .PHONY: lint
 
 # Fix code style
 fix:
 	sbt scalafmtAll
-	isort python contrib experimental
```

Comment:
Why remove `isort` from `make fix`?

Reply:
I found out `profile = black` for isort seems only works well in simple conditions, but we [customized black](https://github.com/eto-ai/rikai/blob/main/pyproject.toml#L1) a lot, so they just can't be compatible with each other and I can't find out what leads to it. May be use `black` as a forced standard in CI and make `isort` optional is better.